windows/src/desktop/kmshell/util/Keyman.System.KeymanStartTask.pas

```diff
@@ -180,5 +202,26 @@ class procedure TKeymanStartTask.RecreateTask;
   CreateTask;
 end;
 
+class procedure TKeymanStartTask.CleanupAlphaTasks(pTaskFolder: ITaskFolder);
```

Comment:
This is a little sad and I wonder if I really need it; but alpha users will have a bit of a mess in their scheduled tasks if I don't include it. We can probably delete it in 15.0 alpha.

Reply:
Maybe mark as TODO for 15.0 alpha